static/css/styles.scss

```diff
@@ -599,19 +601,19 @@ a.subheading-link {
 .button {
   // FIXME: copied from panel.scss .simplified-edit-button
   display: inline-block;
-  background-color: #5CB85C;
+  background-color: #5cb85c;
   color: #fff;
   font-size: 18px;
   padding: 6px 12px;
   border-radius: 6px;
-  border: 1px solid #4CAE4C;
+  border: 1px solid #4cae4c;
 
   &:hover {
     text-decoration: underline;
   }
 }
 
 .intro-js-highlight-overlay {
-  background: rgba(0,0,0,0);
+  background: rgba(#000, 0);
   border: 2px solid rgba(0, 137, 219, 1);
```

Comment:
We could also probably convert the `rgba(0, 137, 210, 1)` to the Sass shorthand of `rgba(#0089db, 1)` (or even just `#0089db` for that matter).